pkg/resource/deploy/source_eval.go

```diff
@@ -951,6 +951,9 @@ func (rm *resmon) SupportsFeature(ctx context.Context,
 		hasSupport = true
 	case "invokeTransforms":
 		hasSupport = true
+	case "packageReferences":
```

Comment:
One wonders if we should consider calling the feature `parameterizationAndPackageReferences`? Sure a bit wordier but very clear everywhere it is used and (I think) invisible to users anyway?

Reply:
We could, if we want to be more explicit I'm not against that.